src/Regression/Helpers/Client/ChromeClientOptions.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Regression\Helpers\Client;
+
+use HeadlessChromium\Page;
+
+/**
+ * @property bool $leaveOpened
+ * @property string $pageLoadingStep
+ * @property int|null $pageLoadingTimeout
+ * @property bool $skipInterception
+ */
+class ChromeClientOptions
```

Comment:
This looks like an overcomplicated DTO. This object is expected to be created once and never get changed in runtime AFAIK. So, this kind of implementation should be sufficient:
```php
class ChromeClientOptions
{
    public function __construct(
        public readonly bool $leaveOpened = false,
        public readonly string $pageLoadingStep = Page::LOAD,
        public readonly ?int $pageLoadingTimeout = null,
        public readonly bool $skipInterception = false) 
    {}
}
```